ObjectiveGit/GTRemote.m

```diff
@@ -104,7 +104,7 @@ - (NSString *)description {
 + (BOOL)isValidRemoteName:(NSString *)name {
 	NSParameterAssert(name != nil);
 
-	return git_remote_is_valid_name(name.UTF8String) == GIT_OK;
+	return git_remote_is_valid_name(name.UTF8String);
```

Comment:
I'd prefer `return (git_remote_is_valid_name(name.UTF8String) == 1 ? YES : NO);` so there's no chance of a -1 error being reported as "true", and we force-map to "BOOL"-compatible values.